background.html

```diff
@@ -6,10 +6,14 @@
 
     <!-- include background scripts -->
     <script type="text/javascript" src="privly-applications/shared/javascripts/local_storage.js"></script>
+    <script type="text/javascript" src="privly-applications/shared/javascripts/storage.js"></script>
```

Comment:
Sorry but what's stylesheets in the `background.html`? I didn't find stylesheets in the background page.. and there are no errors in my background page console.